monofony/core-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+    "bundles": {
+        "Monofony\\Bundle\\CoreBundle\\MonofonyCoreBundle": ["all"]
+    },
+    "container": {
+        "locale": "en",
+        "email_contact": "contact@example.com",
+        "email_name": "Contact AppName",
+        "email_sender": "no-reply@example.com"
+    },
+    "copy-from-package": {
+        "Recipe/config/": "%CONFIG_DIR%/",
```

Comment:
Unrelated note: 

The `Recipe` directory in your bundle is still autoloaded with PSR-4. You should update the autoload config in composer.json.

Reply:
Yes you're right, I will fix that